dotCMS/src/main/java/com/liferay/util/Encryptor.java

```diff
@@ -158,4 +161,60 @@ public static String encrypt(Key key, String plainText)
 
 	private static final Log _log = LogFactory.getLog(Encryptor.class);
 
-}
\ No newline at end of file
```

Comment:
Codacy found an issue: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)